frontend/src/App/AppUpdatedModalContent.css

```diff
@@ -1,6 +1,7 @@
-.version {
+code {
```

Comment:
Does this scope `code`  styling to this component only? We might need to tweak the Markdown parser to allow for passing in a class for the code blocks.

Styling specific elements like this is less efficient and could result in it affecting other usages.

Reply:
yeah it applies to only this block. I can work on the InlineMarkdown component then :)

Reply:
ok, tweaked InlineMarkdown to allow for a different class for the <code> part of the data passed to it.